mysql/errname.go

```diff
@@ -77,7 +77,7 @@ var MySQLErrName = map[uint16]string{
 	ErrTooLongIdent:                             "Identifier name '%-.100s' is too long",
 	ErrDupFieldName:                             "Duplicate column name '%-.192s'",
 	ErrDupKeyName:                               "Duplicate key name '%-.192s'",
-	ErrDupEntry:                                 "Duplicate entry '%-.192s' for key %d",
+	ErrDupEntry:                                 "Duplicate entry '%-.64s' for key '%-.192s'",
```

Comment:
Good advice, however, it's actually supposed to use ErrDupEntryWithKeyName instead of this error. It's hard to choose here. The Duplicate Entry error code that TiDB has been using all the time seems to be wrong. Change the error code, I am afraid there will be some compatibility problems. This error is too common. I don't know if I should follow the same message as ErrDupEntryWithKeyName. @kennytm